pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.ejs

```diff
@@ -83,7 +83,11 @@
           </div>
           <% } %>
         </div>
-
+        <% if (assessment_text_templated) { %>
+          <div class="card-body bg-light border-top py-3">
+            <%- assessment_text_templated %>
+          </div>
+        <% } %>
```

Comment:
Why do we have this extra block here? Was this previously missing from homeworks?

Reply:
Bingo. Can remove if you'd prefer.